tests/expectations/metrics/test_core.py

```diff
@@ -1769,7 +1908,7 @@ def test_map_compound_columns_unique():
     )
     metrics.update(results)
 
-    # Condition metrics return "negative logic" series.
+    # Condition metrics return "negative logic" series
```

Comment:
@NathanFarmer Minor: The comment in the `Pandas` test has the period at the end, so let us keep it here as well.  Thanks.